### PR TITLE
chore: Add digital ocean deployment type

### PR DESCRIPTION
**Pull request by IgorNovitskiy**

### CODE REVIEW

File: packages/create-ship-app/src/scripts/full-stack.sh [hunk 1]
```diff
@@ -105,7 +106,7 @@ cd ..
 
 # Install modules and setup husky
 
-npm install
+# npm install
```
Comment (Oigen43) [inline packages/create-ship-app/src/scripts/full-stack.sh:109]: ?

File: packages/create-ship-app/src/scripts/frontend.sh [hunk 2]
```diff
@@ -12,14 +36,29 @@ cd "$project_name"
...
 cd ship
 git sparse-checkout init --cone
-git sparse-checkout add services/web
+git sparse-checkout add services/web deploy
 git checkout
 cd ../
 
 cp -a ship/services/web/. .
 
+if [ ! -z "$platform_common_dir" ]; then
```
Comment (Oigen43) [inline packages/create-ship-app/src/scripts/frontend.sh:45]: Why not just copying single file file from the workflows folder?
We always use digital ocean apps deployment type for frontend and backend build types. If it evolves in the future, we will update scripts, at the moment this solution looks too tough. You can avoid `installService` and `cleanup.sh `script.

1. copy single workflow file from `workflows` folder
2. remove deploy folder

File: packages/create-ship-app/src/scripts/backend.sh [hunk 3]
```diff
@@ -16,12 +39,19 @@ cd "$project_name"
 git clone --quiet --filter=blob:none --no-checkout --depth 1 --sparse https://github.com/paralect/ship.git
 cd ship
 git sparse-checkout init --cone
-git sparse-checkout add services
+git sparse-checkout add services deploy
```
Comment (Oigen43) [inline packages/create-ship-app/src/scripts/backend.sh:42]: The same comments as for `frontend.sh`

File: packages/create-ship-app/src/index.js [hunk 4]
```diff
@@ -42,7 +42,7 @@ let deploymentType;
   const dockerComposeFileName = utils.getDockerComposeFileName(apiType, dbType);
   const deploymentFolderNames = utils.getDeploymentFolderNames(deploymentType, apiType, dbType);
   
-  await utils.installServices(projectName, buildType, apiType, dbType, dockerComposeFileName, deploymentFolderNames);
+  await utils.installServices(projectName, buildType, deploymentType.replace(/ /g, '_'), apiType, dbType, dockerComposeFileName, deploymentFolderNames);
```
Comment (Oigen43) [inline packages/create-ship-app/src/index.js:45]: why we need this replace? 🤔

**Review by Oigen43 — CHANGES_REQUESTED** [review]

File: packages/create-ship-app/src/index.js (hunk 4 above)
Comment (IgorNovitskiy) [reply packages/create-ship-app/src/index.js:45]: shell splits arguments by space. `Digital ocean apps` will be counted as three separate arguments and replace following arguments

**Pull request merged**